pkg/apis/eventing/v1/trigger_types.go

```diff
@@ -75,8 +75,7 @@ var (
 )
 
 type TriggerSpec struct {
-	// Broker is the broker that this trigger receives events from. If not specified, will default
-	// to 'default'.
```

Comment:
Same comment.

Reply:
same as above, let me know if you want it back.

Reply:
did not add it here as this is v1.